plugin/grepper.vim

```diff
@@ -902,7 +902,11 @@ function! s:run(flags)
     echomsg 'grepper: running' string(cmd)
   endif
 
-  echo printf('Running: %s', s:cmdline)
+  let msg = printf('Running: %s', s:cmdline)
+  if exists('v:echospace') && strwidth(msg) > v:echospace
+    let msg = printf('%.*S...', v:echospace - 3, msg)
```

Comment:
`%.*S` should limit it to display cells (and not bytes like `s`), but it might become too long still.
Looks like a bug in Vim.  Will keep you posted - but not merge this already I'd say.

Reply:
=> https://github.com/vim/vim/pull/4989
Ok to ignore it here though I guess, i.e. this could be merged now already.

Reply:
:+1: